main.py

```diff
@@ -49,6 +74,10 @@ def draw():
 
     for entity in entities:
         WINDOW.blit(entity.sprite.img, entity.pos, entity.sprite.rect)
+        pygame.font.init()
+        myfont = pygame.font.SysFont('Comic Sans MS', 30)
+        textsurface = myfont.render("Blobs: " + str(money), False, (0, 0, 0))
+        WINDOW.blit(textsurface,(700,0))
```

Comment:
do these outside of the for loop, currently it would render and blit the text for every entity we display

Reply:
Done